storagemarket/provider_loop.go

```diff
@@ -59,9 +59,12 @@ func (p *Provider) processDealRequest(deal *types.ProviderDealState) (bool, stri
 	}
 
 	cleanup := func() {
-		errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
+		pub, collat, errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
 		if errf != nil && !xerrors.Is(errf, db.ErrNotFound) {
 			p.dealLogger.LogError(deal.DealUuid, "failed to untag funds during deal cleanup", err)
+		} else {
+			p.dealLogger.Infow(deal.DealUuid, "untagged funds for deal cleanup", "untagged publish", pub, "untagged collateral", collat,
+				"err", errf)
```

Comment:
I think we may want to log something different (or maybe just log nothing at all) if errf is db.ErrNotFound

Reply:
Well, it's be nice to see if it was not found in the db vs there was no error and I don't wanna add one more branch here.